common/utils/StringUtilsTest.cpp

```diff
@@ -561,6 +561,14 @@ void StringUtilsTest::testStringToUInt() {
   OLA_ASSERT_FALSE(StringToInt("1 bar baz", &value, true));
   OLA_ASSERT_FALSE(StringToInt("65537cat", &value, true));
   OLA_ASSERT_FALSE(StringToInt("4294967295bat bar", &value, true));
+
+  //Base36 test
+  OLA_ASSERT_TRUE(StringToInt("12FZ9A", &value, false, 36));
+  OLA_ASSERT_TRUE(StringToInt("12fz9a", &value2, false, 36));
+  OLA_ASSERT_EQ(value, value2);
```

Comment:
Can we also have a test that they evaluate to the numeric value you're expecting please.

Reply:
Perhaps add a false one too and test octal or hex.

Reply:
Added octal, hex and evaluation of expected values.

I did not add false tests, adding "false" test cases to other bases seems like double testing to me, we already tested that the function returns false for overflow and for garbage, there is no reason to expect different behavior under a different base (except for "what is garbage?")

Reply:
Thanks.

I'd just like to see one or two false ones, given especially bases other than octal/hex probably aren't frequently used, so it would be good to know it works as expected on the more obscure OSes please.

Reply:
Done I think

Reply:
Thanks